cli/tests/services.bats

```diff
@@ -29,6 +29,26 @@ project_teardown() {
 
 # ---------------------------------------------------------------------------- #
 
+watchdog_pids_called_with_arg() {
+  # This is a hack to essentially do a `pgrep` without having access to `pgrep`.
+  # The `ps` prints `<pid> <cmd>`, then we use two separate `grep`s so that the
+  # grep command itself doesn't get listed when we search for the data dir.
+  # The `cut` just extracts the PID.
+  pattern="$1"
+  # echo "PATTERN: $pattern" >&3
+  ps_output="$(ps -eo pid,args)"
+  # echo "PS: $ps_output" >&3
+  klauses="$(echo "$ps_output" | grep klaus)"
+  # echo "KLAUSES: $klauses" >&3
```

Comment:
Did you mean to leave these echos in?

Reply:
Yes, that makes it easy to debug if this goes wrong and it's a pain to add it every time